paimon-core/src/main/java/org/apache/paimon/operation/TagDeletion.java

```diff
@@ -29,22 +29,16 @@
 import org.apache.paimon.manifest.ManifestFileMeta;
 import org.apache.paimon.manifest.ManifestList;
 import org.apache.paimon.utils.FileStorePathFactory;
-import org.apache.paimon.utils.SnapshotManager;
-import org.apache.paimon.utils.TagManager;
 
 import javax.annotation.Nullable;
 
-import java.util.HashMap;
 import java.util.HashSet;
-import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
 import static org.apache.paimon.operation.DeletionUtils.addMergedDataFiles;
-import static org.apache.paimon.operation.DeletionUtils.collectManifestSkippingSet;
 import static org.apache.paimon.operation.DeletionUtils.containsDataFile;
 import static org.apache.paimon.operation.DeletionUtils.readEntries;
-import static org.apache.paimon.operation.DeletionUtils.tryDeleteDirectories;
 
 /**
  * Delete tag files. This class doesn't check changelog files because they are handled by {@link
```

Comment:
Maybe we can try to create a abstract class for TagDeletion and SnapshotDeletion.